gprofiler/profilers/java.py

```diff
@@ -830,7 +836,7 @@ def _is_jvm_profiling_supported(self, process: Process) -> bool:
                 return False
         else:
             if self._simple_version_check and process_basename == "java":
-                java_version_output = get_java_version(process, self._stop_event)
+                java_version_output = get_java_version_logged(process, self._stop_event)
```

Comment:
Don't we want the version to be logged even if no checks at all?

If so - we can log the java version from outside this function if `basename == 'java'`

Reply:
I don't want to complicats the logic any more. This is seldom used anyway